recipes/dipy/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
+if errorlevel 1 exit 1
```

Comment:
I think it is OK to remove this file now. Latest conda can use the `script` field on both Windows and *nix.